README.md

```diff
@@ -106,11 +115,10 @@ push git commits and tags, and push the `.gem` file to [rubygems.org][].
 
 ## Copyright
 
-Copyright (c) 2018 SensorTower Inc.<br>
+Copyright (c) 2018-23 SensorTower Inc.<br>
 See [LICENSE.md][] for further details.
 
 
-[travis.ci]: http://travis-ci.org/sensortower/sidekiq-ultimate
 [rubygems.org]: https://rubygems.org
 [LICENSE.md]: https://github.com/sensortower/sidekiq-ultimate/blob/master/LICENSE.txt
-[sidekiq-throttled]: http://travis-ci.org/sensortower/sidekiq-throttled
+[sidekiq-throttled]: https://github.com/ixti/sidekiq-throttled-tmp
```

Comment:
I'd just link to the rubygems page to avoid the tmp renaming that will soon be changed